ast/builtins.go

```diff
@@ -781,6 +784,17 @@ var Sprintf = &Builtin{
 	),
 }
 
+// NumBytes converts strings like 10GB, 5K, 4mb, and the like into an integer number of bytes.
+var NumBytes = &Builtin{
+	Name: "numbytes",
```

Comment:
I'm agnostic on the name and more than happy to entertain suggestions. Totally agree that `parse_*` is preferred here. My reservation about `parse_units` is that the other `parse_*` functions are named based on what they're attempting to extract, e.g. `the parse_certificates` function extracts certificates from a string. In this case, the units aren't being extracted but rather the number of bytes, i.e. I'm trying to get 1048576 `1MB`, not "megabytes."

Reply:
Hmm, that is a good point. Then maybe we go with `parse_bytes`?

One angle we could take too is making it a little bit more generic like `parse_number` and not support the `b`/`B` specifier explicitly.

Reply:
@patrick-east I've decided to go with `parse_num_bytes` for now but happy to be persuaded otherwise. To me, `parse_bytes` suggests that bytes are being parsed. Okay, I see what you mean by the rabbit hole now...

Reply:
We should probably namespace the built-in so something like `units.parse_bytes` feels right to me. If there are no objections to `units.parse_bytes` let's just go with that.